data/battle_anim_scripts.s

```diff
@@ -2812,10 +2812,10 @@ Move_MINIMIZE:
 Move_METRONOME:
 	loadspritegfx ANIM_TAG_FINGER
 	loadspritegfx ANIM_TAG_THOUGHT_BUBBLE
-	createsprite gUnknown_08593114, ANIM_ATTACKER, 11, 0, 100
+	createsprite gThroughtBubbleSpriteTemplate, ANIM_ATTACKER, 11, 0, 100
```

Comment:
Small typo here, there is an extra `r`.  Should be `gThoughtBubbleSpriteTemplate`